apps/pie-storybook/stories/pie-tag.stories.ts

```diff
@@ -49,6 +50,13 @@ const tagStoryMeta: TagStoryMeta = {
                 summary: false,
             },
         },
+        isDimmed: {
+            description: 'If `true`, sets the tag opacity to 50%',
```

Comment:
I think this description is very implementation specific and would require an update if we changed to 55% etc. I would perhaps be a little less specific in this description

Reply:
i used the same description used in the documentation site, could you please suggest something else?

Reply:
"If `true`, lowers the tag opacity.` or something. I don't think we have to copy their description

Reply:
updated 🤝